templates/Deepcell_Postprocessing.ipynb

```diff
@@ -130,11 +131,12 @@
    "source": [
     "# get input data for overlay\n",
     "input_data_xr = data_utils.load_imgs_from_multitiff(deepcell_input_dir, multitiff_files=points_input)\n",
+    "overlay_channels = [input_data_xr.channels.values[:2]]\n",
     "\n",
-    "for fov in input_data_xr.fovs:\n",
-    "    plot_utils.plot_overlay(segmentation_labels.loc[fov, :, :, \"whole_cell\"].values,\n",
-    "                            input_data_xr.loc[fov, :, :, :].values,\n",
-    "                            path=os.path.join(label_dir, f'{fov.values}_overlay.tif'))"
+    "segmentation_utils.visualize_segmentation(\n",
+    "            segmentation_labels_xr=segmentation_labels,\n",
+    "            fovs=input_data_xr.fovs.values, channel_data_xr=input_data_xr,\n",
+    "            chan_list = overlay_channels[0], output_dir=label_dir)"
```

Comment:
Since this is a visualization, it may be best to upload a notebook that shows a sample run with the resulting overlay image so that people can kind of expect what it may look like, feel free to agree or disagree @ackagel and @ngreenwald.

Reply:
agreed, just be sure to remove warnings from the output and just leave relevant figures on the next push.

Reply:
Yeah, this is a good idea. There should be two examples: one that plots the overlay of a single FOV with a single set of overlays in the cell of the notebook. The second showing how multiple overlays can be specified, across multiple FOVs, and these will get saved as TIFs